x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/ShrinkAction.java

```diff
@@ -29,14 +29,14 @@
  * A {@link LifecycleAction} which shrinks the index.
  */
 public class ShrinkAction implements LifecycleAction {
+    private final Logger logger = LogManager.getLogger(ShrinkAction.class);
```

Comment:
Super minor, but this can be `static` (it looks like it used to be but was dropped when it was moved)